src/app/components/badges/BadgeDisplay.tsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import Image from 'next/image';
 import styles from './badgeDisplay.module.css';
-import bg from '../../images/bg.svg';
 import goldBadge from '../../images/goldBadge.svg';
+import silverBadge from '../../images/silverBadge.svg';
+import bronzeBadge from '../../images/bronzeBadge.svg';
 
 export default function BadgeDisplay() {
   return (
-    <div className={styles.badgeContainer}>
-      <Image src={bg} alt='Background' />
-
-      <div className={styles.badgeOverlay}>
+    <div className={styles.badgeWrapper}>
+      <div className={styles.gold}>
         <Image src={goldBadge} alt='Gold Badge' />
       </div>
+      <div className={styles.silver}>
+        <Image src={silverBadge} alt='Gold Badge' />
+        <Image src={bronzeBadge} alt='Gold Badge' />
```

Comment:
I think this can be silver and bronze

Reply:
The styles class name can also be different

Reply:
> The styles class name can also be different

Not sure what do you mean by this.  I changed class Name silver to silverAndBronze reflecting the above comment